src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor

```diff
@@ -85,4 +86,62 @@
         </ExamplePreview>
     </ComponentExampleBox>
 
-</ComponentDemo>
\ No newline at end of file
+
+    <ComponentExampleBox Title="ChoiceGroup with custom options" HTMLSourceCode="@example8HtmlCode" CSharpSourceCode="@example8CSharpCode" ExampleId="example8">
+        <ExamplePreview>
+            <div class="example-desc">ChoiceGroup options are customized by LabelTemplate property in the ChoiceGroupOption class</div>
+            <BitChoiceGroup Label="Pick one" Options="Example8Options">
+            </BitChoiceGroup>
+        </ExamplePreview>
+    </ComponentExampleBox>
+
+</ComponentDemo>
+
+
+@code {
```

Comment:
Plz, Move the code block to the `BitChoiceGroupDemo.razor.cs` file

Reply:
We need to write it to the code block in order to be able to set the HTML value of the RenderFragment property.
like this : [https://github.com/bitfoundation/bitframework/blob/develop/src/Client/Web/Bit.Client.Web.BlazorUI/Components/Nav/BitNav.razor](url)

Reply:
since assigning RenderFragment is very hard in a .cs file, we are forced to use the @code block in a .razor file to assign a proper value.